VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-sounds.service.ts

```diff
@@ -10,12 +10,12 @@ export class NotificationSoundsService {
     hearingStartingAnnounced: boolean;
     currentPlayCount: number;
     hearingAlertSound: HTMLAudioElement;
-
     hearingAlertPlayCount: number;
 
-    initConsultationRequestRingtone(): void {
+    async initConsultationRequestRingtone() {
```

Comment:
is there any reason to remove the return signature? Just curious.

Reply:
I noticed this was the only one to have it in the method signature. TypeScript also makes it easy to infer what the returning type is